crates/udd-sketch/src/lib.rs

```diff
@@ -47,8 +47,8 @@ impl SketchHashKey {
         use SketchHashKey::*;
 
         match *self {
-            Negative(x) => Negative(if x < 0 {x-1} else {x} /2),
-            Positive(x) => Positive(if x < 0 {x-1} else {x} /2),
+            Negative(x) => Negative(if x > 0 {x+1} else {x} /2),
+            Positive(x) => Positive(if x > 0 {x+1} else {x} /2),
```

Comment:
could you explain the change?

Reply:
Certainly, the problem here is that I was combining bins the wrong direction...so I was pairing bins like this: ..., (-2, -1), (0, 1), (2, 3), ... when it should have been ..., (-3, -2), (-1, 0), (1, 2), ...

I've updated the comment on this function indicating the order of combination.